src/dataframerow/dataframerow.jl

```diff
@@ -111,8 +111,10 @@ for T in (:AbstractVector, :Regex, :Not, :Between, :All, :Colon)
         end
 
         if v isa AbstractDict
```

Comment:
Same here (but GitHub won't let me make a suggestion...).

Reply:
I accepted the suggestions and cleaned this proposal, though I am not sure if it is more readable (but I am OK with it).

Reply:
Yeah it's not really more readable, but having nested `if`s can be confusing IMHO, as they look like some code will be run outside of inner `if`s.